tasks/xgettext.js

```diff
@@ -228,14 +242,15 @@ module.exports = function(grunt) {
         extractDirectiveStrings('"', func);
       });
 
-      return messages;
+      return [messages, messageToFilesMap];
     },
 
     handlebars: function(file, options) {
```

Comment:
Use a default value for options as {} or a const options = options || {} so there isn't a possible reference error on options.namespaceSeparator.